vis.mac

```diff
@@ -78,11 +78,13 @@
 #/DarkRate/SetConvert 1.119 #For HPDs
 #/DarkRate/SetConvert 1.126 #For Box and Line PMTs
 
+##Comment out lines the create, open OGLSX, zoom, viewpointThetaPhi, and upVector lines below if using RayTracer
+
 /vis/scene/create
 /vis/open OGLSX
 # You can also set the size and position of the window if you like
 #/vis/open OGLSX 1000x1000-0+0
-/vis/ogl/set/printSize 1000 1000
+#/vis/ogl/set/printSize 1000 1000
```

Comment:
Looking at the diff, this line should be uncommented if we want it to be the same as before this pull request.